crates/miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -189,6 +180,8 @@ const.ACCT_PROCEDURES_CALL_TRACKING_OFFSET=2084
 const.NUM_ACCT_STORAGE_SLOTS_OFFSET=2340
 const.ACCT_STORAGE_SLOTS_SECTION_OFFSET=2344
 
+const.ACCT_NONCE_PTR=ACCT_ID_AND_NONCE_OFFSET+3
```

Comment:
This is an offset, not a pointer - right? If so, we should rename it to `ACCT_NONCE_OFFSET`.

But also, it seems like this is obsolete and we wanted to replace it with `ACCT_NONCE_OFFSET` from line 172 above (which is currently unused).